commcare/src/main/java/org/motechproject/commcare/domain/report/ReportDataInfo.java

```diff
@@ -0,0 +1,50 @@
+package org.motechproject.commcare.domain.report;
+
+
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Represents a single CommCareHQ report data. It's a part of the MOTECH model.
+ */
+
+public class ReportDataInfo {
```

Comment:
It is prepared to use exactly the same way like similarly class in Cases part in CommCare model and can be useful in future actions.